### DISCUSSION TITLE
Add the ability to select ALL available upscalers.

### DISCUSSION THREAD

**Discussion by StanleyPain** [AIrjen/OneButtonPrompt#58] [Ideas]
Could we get the option to select all the available upscalers?

Currently my list looks a bit light.

![image](https://github.com/AIrjen/OneButtonPrompt/assets/132229765/b695da45-0b8e-4f24-b0de-e5b98126a184)

**Comment by AIrjen** [comment]
Hi!

are you using vladmantic? There is a known compatibility issue: https://github.com/AIrjen/OneButtonPrompt/issues/49

basically, you need to reload vladmantic once after starting to make sure all upscalers are loaded in.


Hope this helps!

**Reply by StanleyPain** [reply]
Using A1111's webui. Reloading doesn't seem to fix it. 

Initially I thought it was an extension causing it but I disabled all but the default ones but it didn't work either. :(


edit: changed non-default to default.

**Reply by AIrjen** [reply]
Hi! I've had time to update to WebUI 1.4 and I can reproduce the issue.
It looks like the same issue as what happens in Vladmantic is now happening in WebUI 1.4 as well.

I am able to work around the issue by reloading (go to settings > press "Reload UI"). 

This is going on the to-do list. I do need to figure out how I can resolve this. I might need to rewrite on how gathering the upscalers actually work.

**Reply by StanleyPain** [reply]
Good luck! :)

**Reply by AIrjen** [reply]
A first fix has been deployed. It's not perfect, but it's currently better than nothing. On first pass it seems to load only what is installed.
So the workaround to reload: reloading (go to settings > press "Reload UI"). might still be needed.

Will investigate further.

**Reply by AIrjen** [reply]
Another fix has been put into place. Tested on both WebUI 1.4 and Vlad. Should be workable again!.
